cmd/admin-revoker/main.go

```diff
@@ -126,7 +126,7 @@ func revokeByReg(regID int, reasonCode int, deny bool, cac rpc.CertificateAuthor
 	}
 
 	var certs []core.Certificate
-	_, err = tx.Select(certs, "SELECT serial FROM certificates WHERE registrationID = :regID", map[string]interface{}{"regID": regID})
+	_, err = tx.Select(&certs, "SELECT serial FROM certificates WHERE registrationID = :regID", map[string]interface{}{"regID": regID})
```

Comment:
Since `RegistrationID` isn't a primary key I don't think we can use `Get` here. The other statement could totally use it though!